src/pages/Home/Home.jsx

```diff
@@ -38,9 +39,18 @@ const entries = [
 ];
 
 const Home = () => {
+
+  const [randint, setRandom] = useState(0);
+  
+  useEffect(() => {
+    setRandom(Math.floor(Math.random() * 100 + 1));
```

Comment:
I'm pretty sure the instructions asked for a random number 1 to 100, inclusive, while this (correct me if I'm wrong) generates a random number 1 to 99 inclusive.